artifactory/cli.go

```diff
@@ -3351,6 +3351,7 @@ func createBuildConfiguration(c *cli.Context) *utils.BuildConfiguration {
 		buildNumberArg = ""
 	}
 	buildConfiguration.BuildName, buildConfiguration.BuildNumber = utils.GetBuildNameAndNumber(buildNameArg, buildNumberArg)
+	buildConfiguration.Project = c.String("project")
```

Comment:
Why not getting the env default value here? similar to the build name and number ? (Instead of doing it separately  in the core logic?)

Reply:
@talarian1 - I'll see if I can make this change in a follow up PR.